kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java

```diff
@@ -351,4 +351,40 @@ private InternalClusterMetrics collectTopicsMetrics(Map<String,InternalTopic> to
                 })
             );
     }
+
+    public Mono<Map<String, BigDecimal>> getOffsets (AdminClient ac, KafkaCluster c) {
```

Comment:
What is String? TopicPartition?

Reply:
Topic-Partition-max/min to avoid creating new object in swagger